### PR TITLE
Created UI view for the contact detail

### PR DESCRIPTION
**Pull request by CSaavedra96**
## Description
UI screen for the contact details selected

## Fixes #167799262
https://www.pivotaltracker.com/story/show/167799262

## Type of change
- [ ] Bug fix (non-breaking change which fixes an issue)
- [X] New feature (non-breaking change which adds functionality)

## Additional notes:
None

### CODE REVIEW

**Review by maumaigre — APPROVED** [review]

File: ui/web/src/views/ContactDetails.vue [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+      birthday: "01/01/1990",
+      office_phone: "(664) 111-1111",
+      fax: "(664) 222-2222",
+      tags: []
+    }
+  })
+};
+</script>
+
+
```
Comment (MiguelDiazArkus) [inline ui/web/src/views/ContactDetails.vue]: Just leave one empty line between script and style tags

**Review by MiguelDiazArkus — CHANGES_REQUESTED** [review]
@CSaavedra96 see comment in ContactDetails.vue file.

File: ui/web/src/views/ContactDetails.vue (hunk 1 above)
Comment (CSaavedra96) [reply ui/web/src/views/ContactDetails.vue]: Fixed

**Review by MiguelDiazArkus — APPROVED** [review]
looks good.

**Pull request merged**